README.md

```diff
@@ -0,0 +1,30 @@
+# Connect-4 v1.0
+
+### by Leo Chan, (c)1997
+
+![screenshot](screenshot.png)
+
+### Source information
+* File: Connect4.zip
+
+* Author: Leo Chan
+
+* Release: 1.0 (December 14, 1997)
+
+* Compatibility: PR 2
+
+* Location: contrib/games
+
+* Description: A simple game of Connect 4
+
+* Notes: The project for this game is in the Project directory.
+
+
+
+Please mail any comments to leochan@gpu.srv.ualberta.ca
+
+This is the game of Connect-4.  The game is played by players taking turns dropping pieces down a column.  The first player to get four consecutive pieces in a row, column, or diagonal wins the game.
```

Comment:
As that email address is almost guaranteed to be dead, I'd just keep the screenshot and this paragraph.